packages/@aws-cdk/aws-lambda/lib/runtime.ts

```diff
@@ -135,15 +131,13 @@ export class Runtime {
 
   /**
    * The .NET Core 1.0 runtime (dotnetcore1.0)
-   *
-   * @deprecated Use {@link DOTNET_CORE_2_1}
+   * Legacy runtime no longer supported by AWS Lambda.
```

Comment:
We don't want to remove it?

Reply:
Generally, customers should be able to move their code to v2 and continue using 'legacy runtimes' in their stack if the lambda service allows it. 
The runtime being deprecated by lambda is not related to API deprecations by the CDK. They feel fundamentally different.